deployment/v3/external/object-store/minio/install.sh

```
@@ -18,6 +18,13 @@ function installing_minio() {
   helm -n minio install minio mosip/minio \
   --set image.repository="mosipid/minio" \
   --set image.tag="2022.2.7-debian-10-r0" \
+ALLOW_INSECURE_IMAGES="${ALLOW_INSECURE_IMAGES:-false}"
+
+  helm -n minio install minio mosip/minio \
+  --set image.repository="mosipid/minio" \
+  --set image.tag="2022.2.7-debian-10-r0" \
+  --set global.security.allowInsecureImages="${ALLOW_INSECURE_IMAGES}" \
+  -f values.yaml --version 10.1.6
   -f values.yaml --version 10.1.6
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)